examples/apps/echo-server/src/Server.flix

```diff
@@ -0,0 +1,41 @@
+mod Server {
+    import java.nio.charset.StandardCharsets
+
+    use Peekable.readLine
+    use Writable.writeLine
+
+    pub def launch(ipAddr: IpAddr, port: Int16): Result[IoError, Unit] \ TcpBindWithResult + TcpAcceptWithResult + Logger + Sys + Net + IO =
```

Comment:
Ideally, we'd only have abstract effects here, then all the primitive effects show up in main (potentially via some handler).

I guess Sys is from some of line separator stuff we saw before. What's causing the `Net` effect?

Reply:
(Ditto for client)

Reply:
it was not needed. removed